buildenv/jenkins/build

```diff
@@ -0,0 +1,229 @@
+                    OPENJDK_BRANCH = 'openj9'
+                }
+                try {
+                    if ( OPENJDK_SHA == "" ) {
+                        OPENJDK_SHA = ''
+                    }
+                } catch(ex) {
+                    OPENJDK_SHA = ''
+                }
+                try {
+                    if ( OPENJ9_REPO != "" ) {
+                        OPENJ9_REPO_OPTION = "-openj9-repo=${OPENJ9_REPO}"
+                    } else {
+                        OPENJ9_REPO_OPTION = ''
+                    }
+                } catch(ex) {
+                    OPENJ9_REPO_OPTION = ''
+                }
+                try {
+                    if ( OPENJ9_BRANCH != "" ) {
+                        OPENJ9_BRANCH_OPTION = "-openj9-branch=${OPENJ9_BRANCH}"
+                    } else {
+                        OPENJ9_BRANCH_OPTION = ''
+                    }
+                } catch(ex) {
+                    OPENJ9_BRANCH_OPTION = ''
+                }
+                try {
+                    if ( OPENJ9_SHA != "" ) {
+                        OPENJ9_SHA_OPTION = "-openj9-sha=${OPENJ9_SHA}"
+                    } else {
+                        OPENJ9_SHA_OPTION = ''
+                    }
+                } catch (ex) {
+                    OPENJ9_SHA_OPTION = ''
+                }
+                try {
+                    if ( OMR_REPO != "" ) {
+                        OMR_REPO_OPTION = "-omr-repo=${OMR_REPO}"
+                    } else {
+                        OMR_REPO_OPTION = ''
+                    }
+                } catch(ex) {
+                    OMR_REPO_OPTION = ''
+                }
+                try {
+                    if ( OMR_BRANCH != "" ) {
+                        OMR_BRANCH_OPTION = "-omr-branch=${OMR_BRANCH}"
+                    } else {
+                        OMR_BRANCH_OPTION = ''
+                    }
+                } catch(ex) {
+                    OMR_BRANCH_OPTION = ''
+                }
+                try {
+                    if ( OMR_SHA != "" ) {
+                        OMR_SHA_OPTION = "-omr-sha=${OMR_SHA}"
+                    } else {
+                        OMR_SHA_OPTION = ''
+                    }
+                } catch(ex) {
+                    OMR_SHA_OPTION = ''
+                }
+            }
+
+            git branch: "${OPENJDK_BRANCH}", url: "${OPENJDK_REPO}"
+            sh "git checkout ${OPENJDK_SHA}"
+            sh "bash ./get_source.sh ${OPENJ9_REPO_OPTION} ${OPENJ9_BRANCH_OPTION} ${OPENJ9_SHA_OPTION} ${OMR_REPO_OPTION} ${OMR_BRANCH_OPTION} ${OMR_SHA_OPTION}"
+        }
+    }
+}
+
+def checkout_pullrequest() {
+    stage('Checkout Pull Request') {
+        timestamps {
+            echo 'Look for Dependent Changes'
+            script {
+                def KEYWORD = "depends"
+                def COMMENT = ghprbCommentBody.toLowerCase(Locale.US);
+                def OJDK_PR
+                def OMR_PR
+                omr_bool     = false
```

Comment:
don't indent variables by spacing.  The next time you add a variable, the spacing will be off and instead of a diff of 1 line, you'll need to update all of them.